selfdrive/test/helpers.py

```diff
@@ -72,3 +72,10 @@ def wrap(*args, **kwargs):
 
 def noop(*args, **kwargs):
   pass
+
+
+def read_segment_list(segment_list_path):
+  with open(segment_list_path, "r") as f:
+      seg_list = f.read().splitlines()
+
+  return [(platform[2:], segment) for platform, segment in zip(seg_list[::2], seg_list[1::2], strict=True)]
```

Comment:
wrong indentation

Reply:
Hmm how is there no rule for this...

Reply:
no clue... parameterizing https://docs.astral.sh/ruff/rules/indentation-with-invalid-multiple/ doesn't even work properly with pyproject, from what I've tried. plus 6 is a multiple of 2...